hilla-react/pom.xml

```diff
@@ -0,0 +1,181 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.vaadin</groupId>
+        <artifactId>vaadin-platform-parent</artifactId>
+        <version>23.3-SNAPSHOT</version>
+    </parent>
+    <groupId>dev.hilla</groupId>
+    <artifactId>hilla-react</artifactId>
+    <packaging>jar</packaging>
+    <name>Hilla Platform for React</name>
+    <description>Hilla Platform for React-based Applications</description>
+    <version>${hilla.version}</version>
+    <url>https://hilla.dev</url>
+    <distributionManagement>
+        <repository>
+            <id>vaadin-prereleases</id>
+            <url>https://maven.vaadin.com/vaadin-prereleases/</url>
+        </repository>
+    </distributionManagement>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>dev.hilla</groupId>
+                <artifactId>hilla-bom</artifactId>
+                <version>${project.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>dev.hilla</groupId>
+            <artifactId>endpoint</artifactId>
+            <version>${hilla.version}</version>
```

Comment:
you can remove this `${hilla.version}` here, as the version should come from the hilla-bom. 
same with the `engine-runtime` artifacts.

Reply:
I'll add `endpoint` to the bom.